packages/cardano-services/src/ChainHistory/DbSyncChainHistory/queries.ts

```diff
@@ -37,31 +37,14 @@ export const findTxCollateralsByIds = `
 	WHERE tx.id = ANY($1)
 	ORDER BY tx_in.id ASC`;
 
-export const findTxInputsByAddresses = `
-  ${selectTxInput()}
-	JOIN block ON tx.block_id = block.id
-  WHERE tx_out.address = ANY($1)
-	AND block.block_no >= $2
-	AND block.block_no <= $3
-	ORDER BY tx_in.id ASC`;
-
 export const findTxOutputsByIds = `
   	${selectTxOutput()}
   	WHERE tx.id = ANY($1)
 	ORDER BY tx_out.id ASC`;
 
-export const findTxOutputsByAddresses = `
-  ${selectTxOutput()}
-	JOIN block ON tx.block_id = block.id
-  WHERE tx_out.address = ANY($1)
-	AND block.block_no >= $2
-	AND block.block_no <= $3
-	ORDER BY tx_out.id ASC`;
-
 export const findCollateralOutputsByTxIds = `
 	${selectTxOutput(true)}
-	WHERE tx.id = ANY($1)
-  ORDER BY tx_out.id ASC`;
+	WHERE tx.id = ANY($1)`;
```

Comment:
No need to add sorting in the js code? FYI: even if query seems to return sorted results all the time, one can not rely on that with postgres